### DISCUSSION TITLE
BVM02 and BM6?

### DISCUSSION THREAD

**Discussion by croldan72** [KrystianD/bm2-battery-monitor#12] [Q&A]
Will this work with just the BM2 devices? I have a BVM02 and just discovered this solution and thinking of also using with HA. Wondering if I should return the BVM and just get a BM2.

**Comment by KrystianD** [comment]
No idea - feel free to test it, but I doubt it would work.

**Comment by sid3windr** [comment]
I kind of assumed it would work with the BM6 I just got, but it doesn't look like it does. To the reverse engineering mobile...
